### PR TITLE
Adds a number of new speechbubble icons.

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
imageadd: Alien queens and praetorians have suitably large speechbubbles.
imageadd: Syndicate cyborgs and syndrones have suitably evil robotic speechbubbles.
Imageadd: Blob mobs have suitably uncomfortable-looking speechbubbles.
imageadd: Holoparasites and guardians have suitably robotic and magical speechbubbles.
imageadd: Swarmers have suitably holographic speechbubbles.
imageadd: Slimes have suitably slimy speechbubbles.
/:cl:

The speechbubble a mob uses is now a mob/living var.

machine
Generic machine speechbubble, used for machines, drones, and mecha.
syndibotsyndibot
Syndicate machine speechbubble, used for syndrones, syndicate borgs, viscerators.
blob
Blob mob speechbubble, used for blob spores and blobbernauts.
holo
Holoparasite speechbubble, for holoparasites.
guardian
Guardian speechbubble, for magical guardians.
swarmerswarmer
Swarmer speechbubble, for swarmers.
slime
Slime speechbubble, for slimes.
alienroyal
Alien royalty speechbubble, used for alien queens and praetorians.

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
fe a t u r e?

**Comment by ChangelingRain** [comment]
SPRITE FEATURES ARE IMMUNE TO THE FREEZE

**Comment by GunHog** [comment]
I love em. :+1:

**Comment by PKPenguin321** [comment]
>25 line additions

"sprite feature"

**Comment by duncathan** [comment]
>inb4 bawhoppen

File: code/modules/mob/living/living_defines.dm [hunk 1]
```diff
@@ -50,6 +50,8 @@
 
 	var/list/say_log = list() //a log of what we've said, plain text, no spans or junk, essentially just each individual "message"
 
+	var/bubble_icon = "default" //what icon the mob uses for speechbubbles
```
Comment (PKPenguin321) [inline code/modules/mob/living/living_defines.dm]: >completely new var in living defines
>sprite feature

Comment (RemieRichards) [inline code/modules/mob/living/living_defines.dm]: Fixes include new vars and sometimes entire new systems.
By your logic they are features...

Comment (PKPenguin321) [inline code/modules/mob/living/living_defines.dm]: its not even a fix though

Comment (ChangelingRain) [inline code/modules/mob/living/living_defines.dm]: Pah, removing hardcoding in favor of a var is a fix.

Comment (RemieRichards) [inline code/modules/mob/living/living_defines.dm]: I wasn't saying that, I was stating how your point is stupid.

Comment (PKPenguin321) [inline code/modules/mob/living/living_defines.dm]: isn't the point of a feature freeze to minimize any potentially bug causing additions though
I mean i have no problem with this feature but i just don't think its in the spirit of the freeze
freezes are stupid btw

File: code/game/gamemodes/miniantags/bot_swarm/swarmer.dm [hunk 2]
```diff
@@ -59,7 +60,6 @@
 	speed = 0
 	faction = list("swarmer")
 	AIStatus = AI_OFF
-	projectiletype = /obj/item/projectile/beam/disabler
```
Comment (KorPhaeron) [inline code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: ?

Comment (GunHog) [inline code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: <b>STEALTH NERF!!!!!!!!!!!!!!!!!!!!!</b>

Comment (ChangelingRain) [inline code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: See line 66.

**Comment by WJohn** [comment]
These looks pretty good. :+1:

File: code/game/gamemodes/miniantags/bot_swarm/swarmer.dm (hunk 2 above)
Comment (Giacom) [inline code/game/gamemodes/miniantags/bot_swarm/swarmer.dm]: It was defined twice? Weird.

**Comment by Razharas** [comment]
Very serious fix to horrible speech bubbles

**Pull request merged**